src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -239,7 +240,7 @@ class DataPointsResource[F[_]](val requestSession: RequestSession[F])
       id,
       inclusiveStart.toEpochMilli.toString,
       exclusiveEnd.toEpochMilli.toString,
-      limit
+      Some(limit.getOrElse(Constants.dataPointsBatchSize))
```

Comment:
`limit.orElse(Some(Constants.dataPointsBatchSize))` is a bit more idiomatic, but this is ok.